bin/utils/normalizeSchema.ts

```diff
@@ -6,6 +6,7 @@ const metaProperties = [
   "$id",
   "title",
   "description",
+  "default",
```

Comment:
"default" is a meta-data annotation? https://json-schema.org/draft/2019-09/json-schema-validation.html#rfc.section.9.2

Reply:
We use `draft-07` iirc
I remembered wrong. Thanks though.
I'm just wondering why you added it, since we don't use it anywhere currently

Reply:
Could you revert this change in this PR and we can revisit this in another PR later on

Reply:
I think we do use "default", specifically in `repository.schema.json` https://github.com/octokit/webhooks/blob/933e092af3deff7c734d1a8da43fb7c33c610036/payload-schemas/schemas/common/repository.schema.json#L158

As a result, `ref-common-schemas.ts` doesn't replace `forkee` with a reference without this `metaProperties` change. So I think it's related to this PR and should be included, but give the word and I'll revert it.